services/keycloak-service-impl/src/test/java/org/kontinuity/catapult/service/keycloak/impl/KeycloakServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package org.kontinuity.catapult.service.keycloak.impl;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * To get a valid token:
+ *
+ * - Open Chrome and go to: http://prod-preview.openshift.io/
+ * - After authentication, grab the token from the URL. There should be a http://prod-preview.openshift.io/?token=XXX (Use the network console if needed)
+ *
+ * @author <a href="mailto:ggastald@redhat.com">George Gastaldi</a>
+ */
+public class KeycloakServiceImplTest {
+    @Test
+    public void testBuildUrl() {
+        String url = KeycloakServiceImpl.buildURL("http://sso.prod-preview.openshift.io", "fabric8", "github");
+        Assert.assertEquals("http://sso.prod-preview.openshift.io/auth/realms/fabric8/broker/github/token", url);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testInvalidTokenOpenshift() {
+        String token = "token";
+        KeycloakServiceImpl service = new KeycloakServiceImpl("http://sso.prod-preview.openshift.io", "fabric8");
+        System.out.println(service.getOpenshiftV3Token(token));
```

Comment:
Not really, just wanted to display the token :) Hopefully the System.out will not be called in this test ;)